tests/test_run_config_measurement.py

```diff
@@ -208,6 +210,43 @@ def test_is_better_than_consistency(self):
         self.assertTrue(self.rcm2.is_better_than(self.rcm3))
         self.assertFalse(self.rcm3.is_better_than(self.rcm2))
 
+    def test_compare_measurements(self):
+        """
+        Test to ensure compare measurement function returns
+        the correct magnitude
+        # RCM4's throughput is 1000
+        # RCM5's throughput is 2000
+        # Therefore, the magnitude is (RCM5 - RCM4) / avg throughput
+        #                             (2000 - 1000) / 1500
+        """
+        rcm4_vs_rcm5 = self.rcm4.compare_measurements(self.rcm5)
+        self.assertEqual(rcm4_vs_rcm5, 1000 / 1500)
+
+    def test_is_passing_constraints_none(self):
+        """
+        Test to ensure constraints are reported as passing
+        if none were specified
+        """
+        self.assertTrue(self.rcm5.is_passing_constraints())
+
+    def test_is_passing_constraints(self):
+        """
+        Test to ensure constraints are reported as
+        passing/failing if model is above/below
+        throughput threshold
```

Comment:
Should we also check for max latency?

Reply:
This isn't needed. This functionality is covered in test_constraint_manager().